packages/cadl-ranch-specs/http/clients/interfaces/main.cadl

```diff
@@ -32,7 +32,6 @@ interface dogs {
 
   @scenario
   @scenarioDoc("Illustrate grouping operations on subclient.")
-  @route("/models")
```

Comment:
why delete @route?

Reply:
It looks like added wrong, because other operations don't have this.